### ISSUE TITLE
IOU - Requesting money badge in the DM disappears after tap on "Message..." option in Detail Page

### ISSUE THREAD

**Issue by kbecciv** [Expensify/App#9200]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
Precondition:
Complete the request money flow and send payment request to another account you have access to.
1. Log-in to the other account
2. Go to chat conversation where you owed the money
3. Check the Requesting money badge in the DM
4. Tap on user's avatar on top of the chat
5. Tap Message user
6. Check the Requesting money badge in the DM

## Expected Result:
Requesting money badge in the DM does not disappear after tap on "Message..." option in Detail Page

## Actual Result:
Requesting money badge in the DM disappears after tap on "Message..." option in Detail Page

## Workaround:
Unknown

## Platform:

Where is this issue occurring?

- Web
- iOS
- Android
- Desktop App
- Mobile Web

**Version Number:** 1.1.68.2

**Reproducible in staging?:** Yes

**Reproducible in production?:** Yes

**Email or phone of affected tester (no customers):** any

**Logs:** https://stackoverflow.com/c/expensify/questions/4856

**Notes/Photos/Videos:** Any additional supporting documentation

Uploading Screen_Recording_20220526-205132_New Expensify.mp4…

**Expensify/Expensify Issue URL:** 

**Issue reported by:** Applause-Internal Team

**Slack conversation:** 

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by JmillsExpensify** [comment]
We're actually going to remove this badge in an upcoming release, so I'm inclined to close this issue and do nothing until the badge is removed from the product. cc @trjExpensify what do you think?

**Comment by iwiznia** [comment]
Yeah, I agree. No sense in fixing something we will be removing shortly
